tests/test_registry.py

```diff
@@ -11,9 +11,9 @@ def _getTargetClass(self):
     def _makeOne(self, *args, **kw):
         return self._getTargetClass()(*args, **kw)
 
-    def test___nonzero__(self):
+    def test___bool__(self):
         registry = self._makeOne()
-        self.assertEqual(registry.__nonzero__(), True)
+        self.assertEqual(bool(registry), True)
```

Comment:
same issue here, it should invoke `__bool__` directly to ensure it's testing what it wants to test - implementing the protocol